src/plugins/console/public/application/components/help_panel.tsx

```diff
@@ -43,9 +44,30 @@ export function HelpPanel(props: Props) {
           <p>
             <FormattedMessage
               id="console.helpPage.requestFormatDescription"
-              defaultMessage="You can type one or more requests in the white editor. Console understands requests in a compact format:"
+              defaultMessage="You can type one or more requests in the editor. Console understands requests in a compact format."
             />
           </p>
+          <p>
+            <FormattedMessage id="console.helpPage.learnAbout" defaultMessage="Learn about" />
```

Comment:
I think `Console` and `Query DSL` should go into `values`.  

See how it was implemented in next case: 
![image](https://user-images.githubusercontent.com/20072247/145837867-3914fb2c-20ba-46ac-b7c5-14d4004600c5.png)